src/main/java/org/takes/facets/fork/MediaType.java

```diff
@@ -151,14 +154,12 @@ private static String highPart(final String text) {
      * @return The low part of the media type.
      */
     private static String lowPart(final String text) {
-        final String sector;
         final String[] sectors = MediaType.sectors(text);
+        Text sector = new TextOf("");
```

Comment:
@fanifieiev `final` ...

Reply:
@paulodamaso Ok, wonder why `qulice`did not raise the issue.